code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -327,6 +324,289 @@ datum/reagent/toxin/staminatoxin/on_mob_life(mob/living/carbon/M)
+		M.adjustBruteLoss(0.2*REM)
+	if(prob(15))
+		M << "You scratch at your leg."
+		M.adjustBruteLoss(0.2*REM)
+	if(prob(15))
+		M << "You scratch at your arm."
+		M.adjustBruteLoss(0.2*REM)
+	if(prob(3))
+		M.reagents.add_reagent("histamine",rand(1,3))
+		M.reagents.remove_reagent("itching_powder",1)
+	..()
+
+datum/reagent/toxin/initropidril
+	name = "Initropidril"
+	id = "initropidril"
+	description = "Causes some toxin damage, 5% chances to cause stunning, suffocation, or immediate heart failure."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	toxpwr = 2.5
+
+datum/reagent/toxin/initropidril/on_mob_life(var/mob/living/M as mob)
+	if(prob(5))
+		var/picked_option = rand(1,3)
+		switch(picked_option)
+			if(1)
+				M.Stun(3)
+				M.Weaken(3)
+			if(2)
+				M.losebreath += 10
+				M.adjustOxyLoss(rand(5,25))
+			if(3)
+				var/mob/living/carbon/human/H = M
+				if(!H.heart_attack)
+					H.visible_message("<span class = 'userdanger'>[H] clutches at their chest as if their heart stopped!</span>")
+					H.heart_attack = 1 // rip in pepperoni
+				else
+					H.losebreath += 10
+					H.adjustOxyLoss(rand(5,25))
+	..()
+
+datum/reagent/toxin/pancuronium
+	name = "Pancuronium"
+	id = "pancuronium"
+	description = "Knocks you out after 30 seconds, 7% chance to cause some oxygen loss."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	toxpwr = 0
+
+datum/reagent/toxin/pancuronium/on_mob_life(var/mob/living/M as mob)
+	if(current_cycle >= 30)
+		M.SetParalysis(3)
+	if(prob(7))
+		M.losebreath += 4
+	..()
+
+datum/reagent/toxin/sodium_thiopental
+	name = "Sodium Thiopental"
+	id = "sodium_thiopental"
+	description = "Puts you to sleep after 30 seconds, along with some major stamina loss."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.75 * REAGENTS_METABOLISM
+	toxpwr = 0
+
+datum/reagent/toxin/sodium_thiopental/on_mob_life(var/mob/living/M as mob)
+	if(current_cycle >= 30)
+		M.sleeping += 1.5
+	M.adjustStaminaLoss(5*REM)
+	..()
+
+datum/reagent/toxin/sulfonal
+	name = "Sulfonal"
+	id = "sulfonal"
+	description = "Deals some toxin damage, and puts you to sleep after 66 seconds."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.125 * REAGENTS_METABOLISM
+	toxpwr = 0.5
+
+datum/reagent/toxin/sulfonal/on_mob_life(var/mob/living/M as mob)
+	if(current_cycle >= 66)
+		M.sleeping += 1.5
+	..()
+
+datum/reagent/toxin/amanitin
+	name = "Amanitin"
+	id = "amanitin"
+	description = "On the last second that it's in you, it hits you with a stack of toxin damage based on how long it's been in you. The more you use, the longer it takes before anything happens, but the harder it hits when it does."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	toxpwr = 0
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+
+datum/reagent/toxin/amanitin/on_mob_delete(var/mob/living/M as mob)
+	M.adjustToxLoss(current_cycle*3*REM)
```

Comment:
REM was already there, it's used for nearly all reagent damage, plus it's equal to 1 by default.